TeleSharp.TL/TLContext.cs

```diff
@@ -14,20 +14,17 @@ public static class TLContext
 
         public static void Init()
         {
-            try
+            Types = new Dictionary<int, Type>();
+            Types = (from t in Assembly.GetExecutingAssembly().GetTypes()
+                        where t.IsClass && t.Namespace.StartsWith("TeleSharp.TL")
+                        where t.IsSubclassOf(typeof(TLObject))
+                        where t.GetCustomAttribute(typeof(TLObjectAttribute)) != null
+                        select t).ToDictionary(x => ((TLObjectAttribute)x.GetCustomAttribute(typeof(TLObjectAttribute))).Constructor, x => x);
+
+            if (!Types.ContainsKey(481674261))
```

Comment:
ah I see what you did here, but actually I think the best approach is simply changing the line `Types.Add(481674261, typeof(TLVector<>));` to simply `Types[481674261] = typeof(TLVector<>);`

Reply:
True -> This is a better method :) Changed